src/python/director/lcmUtils.py

```diff
@@ -113,7 +113,7 @@ def handleMessage(messageData):
                 # except ValueError:
                 #     pass
         if loadSuccessful:
-            callback(msg)
+            callback(msg, channel)
```

Comment:
we'll have to add a new optional arg to addSubscriber() to select whether we should call callback(msg) or callback(msg, channel).  There is a lot of existing code that connects callback functions that only expect a single argument.  Then the drake visualizer code can call addSubscriber(..., channelCallback=True) or something like that maybe?